django/core/tests/base.py

```diff
@@ -115,3 +121,25 @@ def create_unsaved(self, **overrides):
         if password:
             user.set_password(password)
         return user
+
+
+class ChDir:
```

Comment:
this is repeated twice in the codebase. I'd prefer it be removed entirely though since its usage is typically an antipattern in shell scripting..